src/structures.jl

```diff
@@ -47,3 +47,25 @@ _strat_per(t::TimePeriod) = 1
 _opscen(t::TimePeriod) = 1
 _rper(t::TimePeriod) = 1
 _branch(t::TimePeriod) = 1
+
```

Comment:
This change does not fix [PR#6](https://github.com/sintefore/TimeStruct.jl/pull/6). The same issue still occurs, when operating with `strategic_periods(SimpleTimes)`. In fact, it even worsens the situation as now the function `duration` is not defined for the iterator from above., as it is for `strategic_periods(TwoLevel)`. In this situation, it would break code that was previously working.

I would suggest to add:
```julia
duration(ts::SingleTimeStructWrapper) = 1
```
This way, it does not break any code and we still have the desired outcome, that is that if we have a multiplication with `duration(sp)` with `sp isa StrategicPeriod`, then the value is independent of the `_total_duration(SimpleTimes())`.

Reply:
This should now hopefully be fixed by the recent commits. `duration` is replaced with the function `duration_strat` to clarify that the duration is in units of the strategic period. We can set `duration(sp::AbstractStrategicPeriod) = duration_strat(sp)` if needed for backwards compatibility.